testresults/get.go

```diff
@@ -6,47 +6,59 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"strings"
 
 	"github.com/evergreen-ci/timber"
 	"github.com/mongodb/grip"
 	"github.com/pkg/errors"
 )
 
-// TestResultsGetOptions specify the required and optional information to
-// create the test results HTTP GET request to cedar.
-type TestResultsGetOptions struct {
+// Valid sort by keys.
+const (
+	SortByStart      = "start"
```

Comment:
I was going to that originally, but the only value of making them enums is to do validation. Since this is a decoupled client for the REST route, I would rather future proof it and not do validation (in the case that we add another valid sort key and someone is using an outdated timber, their validation would fail even though the sort key is valid).